ci/build/build-standalone-release.sh

```diff
@@ -28,7 +28,7 @@ main() {
   ln -s "./lib/node" "$RELEASE_PATH/node"
 
   pushd "$RELEASE_PATH"
-  yarn --production --frozen-lockfile
+  npm install --unsafe-perm --omit=dev
```

Comment:
```suggestion
  npm ci --unsafe-perm --omit=dev
```

`npm ci` is the equivalent of `yarn --frozen-lockfile` I believe?